libs/documentation/src/lib/storybook/formly/formly-text-area/formly-text-area-description/formly-text-area-description.component.ts

```diff
@@ -0,0 +1,34 @@
+import { AfterViewInit, Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+
+@Component({
+  selector: 'sds-formly-text-area-description',
+  templateUrl: './formly-text-area-description.component.html',
+})
+export class FormlyTextAreaDescriptionComponent {
+  @Input('config')
+  set config(config: object) {
+    const temp = this.fields[0];
+    temp['templateOptions'] = config;
+    this.fields = [temp];
+  }
+
+  form = new FormGroup({});
+  model: any = {};
+  options: FormlyFormOptions = {};
+  fields: FormlyFieldConfig[] = [
+    {
+      key: 'entity',
+      type: 'textarea',
+      templateOptions: {
+        label: 'Entity Description',
+        description: 'Enter the description for your entity.',
+      },
+    },
+  ];
+
+  onModelChange($event) {
```

Comment:
Rather than have this console in every demo, I would say it'd be better to just create an additional demo showing just this output.